_episodes/16-resources.md

```diff
@@ -97,8 +97,7 @@ Some interesting fields include the following:
 
 ## Measuring the statistics of currently running tasks
 
-One very useful feature of SLURM is the ability to SSH to a node where a job is running and check
-how it's doing. To do this, check where a job is running with `squeue`, then run `ssh nodename`.
+Typically, clusters allow users to SSH directly into worker nodes from the head node. This is useful to check on a running job and see how it's doing. To do this, check where a job is running with `squeue`, then run `ssh nodename`.
```

Comment:
let's not use `ssh` as a verb, that is community lingo. Did we introduce compute nodes or worker nodes before? I am afraid, the concept of a head node does not exist in the cloud, right?

Reply:
Suggest " ... clusters allow users to connect directly to worker nodes".  The fact that this operation is via ssh is implicit in the command, which is later in the same sentence.

Reply:
sounds good to me.